h35/src/components/Search.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+
+export default function Search({ data }) {
+  const [ name, setName ] = useState(null); //Store the name to search for
+  const [ cards, setCards ] = useState([]); //Store the cards to display
+
+  const imageURL = 'https://thronesapi.com/assets/images/'; //Image URL string
+  
+  //Issue:  1st time name is searched for, nothing happens - have to click Submit 2x; why?
+  const onSubmit = (event) => {
+    //Get and set the name to search for
+    event.preventDefault();
+    const results = [];
+    const target = event.target.charName.value;
+    setName(target);
+    //Search the data for full and partial matches in the fullName field
+    if(name) {
+      data.forEach((char) => {
+        if (char.fullName.includes(name)) {
```

Comment:
supporting lowercase searches here would be a good addition. I get no search results for "stark" for example since the "s" is lowercase.

Reply:
Updated this functionality.